.travis.yml

```diff
@@ -105,5 +107,5 @@ deploy:
   on:
     tags: true
     repo: MongoEngine/mongoengine
-    condition: ($PYMONGO = 3.x) && ($MONGODB = 3.4.17)
+    condition: ($PYMONGO = ${PYMONGO_3_X}) && ($MONGODB = ${MONGODB_3_4})
```

Comment:
I have a hunch that this will work, but just wanted to see if you've tested it in any way (or verified it with Travis' Support). If not, I guess we could cut a v0.19.0 release ASAP (we have enough changes already) and test it that way.

Technically I *think* you might be able to use [`travis-conditions`](https://docs.travis-ci.com/user/conditions-testing) to test it, but I haven't dug deep enough into it to fully understand how.

Reply:
I must admit that I didn't test it, I gave a shot to `travis-conditions` gem but all I can say is that it accepts environment variable. Now if $PYMONGO and $MONGODB are able to make it to the deploy>on>condition, its likely that the global variable will make it as well. Personally I'd be in favor of waiting a bit more before releasing 0.19.0, simply because people may be afraid of upgrading if there is a big gap in the versions. Just my 2 cents :) I leave the decision up to you if you want to do this now

Reply:
Makes sense (both wrt the conditions and the release) :)